client/app/_components/ui/button/Button.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { Slot } from '@radix-ui/react-slot'
+import { cva, type VariantProps } from 'class-variance-authority'
+import { PiSpinnerGap } from 'react-icons/pi'
```

Comment:
Sorry, I used Phosphor Icons because I could not find a good loading icon in Feather or MaterialUI